plugins/tailwind.ts

```diff
@@ -0,0 +1,52 @@
+import type { Plugin } from "../server.ts";
+import { type Config, tailwindcss as tailwind } from "./tailwind_deps.ts";
+import postcssPlugin, { type PostCssPluginOptions } from "./postcss.ts";
+
+/**
+ * Fresh PostCSS Tailwind Plugin
+ * @param options - {@link PostCssPluginOptions} PostCSS plugin options
+ * @param config - Tailwind configuration
+ * @returns PostCSS plugin to process Tailwind in Fresh
+ * @example
+ * ```ts
+ * import { defineConfig } from "$fresh/server.ts";
+ * import tailwindPlugin from "$fresh/plugins/tailwind.ts";
+ *
+ * export default defineConfig({
+ *   plugins: [
+ *     tailwindPlugin({
+ *       css: ["./src/styles.css"],
+ *     }, (await import("./tailwind.config.ts")).default),
```

Comment:
It'd be more apparent if we just passed the basic config object here. Otherwise, a newcomer might wonder what the config object is supposed to look like. Perhaps, the minimal needed config object:
```ts
{
  content: [
    "./routes/**/*.{ts,tsx}",
    "./islands/**/*.tsx",
    "./components/**/*.tsx",
  ]
}
```

Reply:
Yeah, plus I later realized an `import tailwindConfig from ./tailwind.config.ts` at the top might have made more sense too.